imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -34,16 +34,19 @@ class OrderTable extends Component {
     handleSelect: PropTypes.func,
     multipleSelect: PropTypes.bool,
     orders: PropTypes.array,
+    renderBulkOrderActionsBar: PropTypes.func,
+    renderPaginationClassNameHidden: PropTypes.func,
+    renderTableClassNameHidden: PropTypes.func,
     selectAllOrders: PropTypes.func,
     selectedItems: PropTypes.array,
     shippingBadgeStatus: PropTypes.func
   }
 
   render() {
     const {
-      orders, selectedItems, handleSelect,
-      handleClick, multipleSelect, selectAllOrders,
-      fulfillmentBadgeStatus, shippingBadgeStatus
+      orders, selectedItems, handleSelect, handleClick, multipleSelect, selectAllOrders,
```

Comment:
So I think this might be better to just use `this.props.xxx` rather than this long destructuring statement.